core/validatorapi/teku.go

```diff
@@ -32,10 +32,21 @@ type TekuProposerConfig struct {
 
 type TekuBuilder struct {
 	Enabled   bool              `json:"enabled"`
+	GasLimit  uint              `json:"gas_limit"`
 	Overrides map[string]string `json:"registration_overrides,omitempty"`
 }
 
-const dead = "0x000000000000000000000000000000000000dead"
+var feeRecipient = ""
+
+// var cluster.Lock lock
```

Comment:
Need to uncomment this not sure whats the variable/ file i need to import to gain access to whats in the cluster lock 

saw "github.com/obolnetwork/charon/cluster"

can see it's passed in via 

func Run(ctx context.Context, conf Config) (err error) {

where conf can be used with 

lock, err := loadLock(conf)